### PR TITLE
Flatten Event templates

### PR DESCRIPTION
**Pull request by uakyol**
Templates shouldn't have message fields unless the messages are protobuf well-known types. All template fields should either be a scalar type or a well-known protobuf type (https://protobuf.dev/reference/protobuf/google.protobuf/)

### CODE REVIEW

File: src/main/proto/eventtemplates/display_template.proto
Comment (SanjayVas) [inline src/main/proto/eventtemplates/display_template.proto]: Can this be true when `viewable_0_percent_plus` is false? If not, document the relationship between fields.

File: src/main/proto/eventtemplates/video_template.proto
Comment (SanjayVas) [inline src/main/proto/eventtemplates/video_template.proto]: This comment will be applied only to the field above which it is placed, meaning it applies only to the `completed_0_percent_plus` field.

Each field needs to be documented separately instead.

File: src/main/proto/eventtemplates/video_template.proto
Comment (SanjayVas) [inline src/main/proto/eventtemplates/video_template.proto]: I'm not going to block on this since my understanding is that alignment was difficult to obtain, but I want to call out that it's poor design to allow invariants to be violated like this. A better design would be either a numerical field representing the completion ratio or an enumeration. Both of those options would make it impossible to have a message where "50%+ completed" is true but "25%+ completed" is false.

**Review by SanjayVas — CHANGES_REQUESTED** [review]

File: src/main/proto/eventtemplates/video_template.proto
Comment (SanjayVas) [reply src/main/proto/eventtemplates/video_template.proto]: An enumeration would have the additional benefit of being shareable across templates.

File: src/main/proto/eventtemplates/display_template.proto
Comment (uakyol) [reply src/main/proto/eventtemplates/display_template.proto]: it can be true. These are different standards @mariolamassaavedra keep me honest here please.

File: src/main/proto/eventtemplates/video_template.proto
Comment (uakyol) [reply src/main/proto/eventtemplates/video_template.proto]: Thanks. Hope this covers what we are looking for.

**Review by SanjayVas — APPROVED** [review]

**Review by zasadnyy — APPROVED** [review]
lgtm

**Comment by ghost** [comment]
LGTM ✅

**Comment by uakyol** [comment]
Sorry everyone, @zasadnyy @SanjayVas @dawnwang22 , can I get another lgtm, I added field anotations

**Comment by ghost** [comment]
LG ✅

**Review by mariolamassaavedra — APPROVED** [review]

File: src/main/proto/eventtemplates/display_template.proto
Comment (SanjayVas) [inline src/main/proto/eventtemplates/display_template.proto:28]: The `name` in the `event_template` annotation needs to be lowercase, as it **must** match the field name in the corresponding event message.

This comment is actually for line 27, and also applies to the video template.

**Review by SanjayVas — CHANGES_REQUESTED** [review]

File: src/main/proto/eventtemplates/display_template.proto
Comment (uakyol) [reply src/main/proto/eventtemplates/display_template.proto:28]: thank you. you mean line 26 right? That is the `name` field. 27 is `display_name`

**Review by SanjayVas — APPROVED** [review]

**Pull request merged**
